libnmstate/nm/vxlan.py

```diff
@@ -76,20 +72,5 @@ def get_info(device):
 def _get_destination_port(device):
     """
     Retrieve the destination port.
-
-    The destination port is retrieved from the profile settings instead
-    of the device (which represents the kernel state)
-    due to an existing issue [1].
-
-    [1] https://bugzilla.redhat.com/show_bug.cgi?id=1768388
     """
-    if nm.nmclient.nm_version() >= StrictVersion("1.20.6"):
-        return device.get_dst_port()
-    else:
-        con = connection.ConnectionProfile()
-        con.import_by_device(device)
-        if con.profile:
-            vxlan_settings = con.profile.get_setting_vxlan()
-            if vxlan_settings:
```

Comment:
Could you please drop this workaround in a separate commit?

Reply:
Moved to https://github.com/nmstate/nmstate/pull/1051